index.html

```diff
@@ -272,6 +270,9 @@ <h4>Ready?</h4>
                 <div id="canvas-smooth">
                   <label>Canvas Smoothing <input type="checkbox" id="canvas-smooth-val" /></label>
                 </div>
+                <div id="canvas-smooth">
```

Comment:
It has the same ID as the div above, was it intended?

Reply:
I removed the unecessary divs and ids